docs/frameworks/symfony.md

```diff
@@ -6,90 +6,112 @@ introduction: Learn how to deploy serverless Symfony applications on AWS Lambda
-        events:
-            - httpApi: '*'
-    console:
-        handler: bin/console
-        timeout: 120 # in seconds
-        layers:
-            - ${bref:layer.php-74} # PHP
-            - ${bref:layer.console} # The "console" layer
+  # This function runs the Symfony website/API
+  web:
+    handler: public/index.php
+    timeout: 28 # in seconds (API Gateway has a timeout of 29 seconds)
+    layers:
+      - ${bref:layer.php-80-fpm}
+    events:
+      - httpApi: '*'
+  # This function let us run console commands in Lambda
+  console:
+    handler: bin/console
+    timeout: 120 # in seconds
+    layers:
+      - ${bref:layer.php-80} # PHP
+      - ${bref:layer.console} # The "console" layer
+
+package:
+  patterns:
+    # Excluded files and folders for deployment
+    - '!node_modules/**'
+    - '!tests/**'
+    - '!var/**'
+    # If you want to include files and folders that are part of excluded folders,
+    # add them at the end
+    - 'var/cache/prod/**'
 ```
 
-We still have a few modifications to do on the application to make it compatible with AWS Lambda.
+You still have a few modifications to do on the application to make it compatible with AWS Lambda.
 
-Since [the filesystem is readonly](/docs/environment/storage.md) except for `/tmp` we need to customize where the cache and the logs are stored in the `src/Kernel.php` file. This is done by adding 2 new methods to the `Kernel` class:
+Since [the filesystem is readonly](/docs/environment/storage.md) except for `/tmp` you need to customize where the cache and logs are stored in
+the `src/Kernel.php` file. This is done by extending the default `Kernel` class create by Symfony with the `Kernel` class provide by the bridge:
 
-```php
-    public function getLogDir()
-    {
-        // When on the lambda only /tmp is writeable
-        if (isset($_SERVER['LAMBDA_TASK_ROOT'])) {
-            return '/tmp/log/';
-        }
+```diff
+// src/Kernel.php
 
-        return parent::getLogDir();
-    }
+namespace App;
 
-    public function getCacheDir()
-    {
-        // When on the lambda only /tmp is writeable
-        if (isset($_SERVER['LAMBDA_TASK_ROOT'])) {
-            return '/tmp/cache/'.$this->environment;
-        }
++ use Bref\SymfonyBridge\BrefKernel;
+use Symfony\Bundle\FrameworkBundle\Kernel\MicroKernelTrait;
+use Symfony\Component\Config\Loader\LoaderInterface;
+use Symfony\Component\Config\Resource\FileResource;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+-use Symfony\Component\HttpKernel\Kernel as BaseKernel;
+use Symfony\Component\Routing\RouteCollectionBuilder;
 
-        return parent::getCacheDir();
-    }
+- class Kernel extends BaseKernel
++ class Kernel extends BrefKernel
+{
+    // ...
 ```
 
-We can also separate the [build directory](https://symfony.com/doc/current/reference/configuration/kernel.html#build-directory)
+You can also separate the [build directory](https://symfony.com/doc/current/reference/configuration/kernel.html#build-directory)
```

Comment:
I've tested this several times and this works really bad with the bridge for now (see for example https://github.com/brefphp/symfony-bridge/pull/36 and https://github.com/symfony/symfony/issues/40827)

I would delete this for now as it causes more problems than it solves and the bridge is more performant on projects where I tested both approaches.

Reply:
Fixed